Source/Core/Common/Swap.h

```diff
@@ -166,4 +166,16 @@ inline T FromBigEndian(T data)
   swap<sizeof(data)>(reinterpret_cast<u8*>(&data));
   return data;
 }
+
+template <typename value_type>
+struct BigEndianValue
+{
+  static_assert(std::is_arithmetic<value_type>::value, "value_type must be an arithmetic type");
```

Comment:
`::value` can be `()` (or the variable template `is_arithmetic_v` can be used, if all compilers we support have them).

Reply:
Thanks, I forgot about `()`.